src/Search/Type/Base.php

```diff
@@ -1,9 +1,9 @@
 <?php
-namespace Search\Type;
+namespace Search\Seach\Type;
 
 use Cake\ORM\Query;
 use Cake\Core\InstanceConfigTrait;
-use Search\Manager;
+use Search\Search\Manager;
```

Comment:
I agree that it looks like a "double" but @dereuromark explained it right, it's because of the vendor + sub-folder namespace. Having the search related classes simply thrown into the "root" of the plugin doesn't feel right either.